aws/v12.1.1/README.md

```diff
@@ -0,0 +1,23 @@
+# :zap: Giant Swarm Release v12.1.1 for AWS :zap:
+
+**If you are upgrading from 12.0.0, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades Cert Manager from upstream v0.9.0 to v0.15.2.
+
+It is part of a larger effort to maximize reliability of upgrades, by keeping 3rd party components within 30 days of upstream releases.
+
+**This upgrade will stop Cert Manager from reconciling your existing resources** due to changes in its API - manual intervention is required to update affected resources. While the **negative impact to your workloads is low-to-none**, to minimize disruption, **we recommend discussing this upgrade with your Solution Engineer first**.
```

Comment:
This note is relevant for users upgrading from pre 12.0.0 versions. Not sure should it be repeated in 12.0.1, and if so, maybe in some modified form.

Reply:
@marians just announced that there is no active AWS v12 release, right? So for now, this statement will apply to all? :)

Reply:
Releases that are not active, are not necessarily removed/archived.

Customers upgrading from say 11.5.0 to 12.1.1 are expected to read all the release notes in between, even deprecated ones. Just because they should, doesn't mean they will. To increase likelihood not to have such important information missed, we sometimes repeat notes across multiple patch releases.

Customer could already have cluster upgraded to 12.1.0 and have sorted out the cert-manager upgrade in the process, so this note would not apply to them.
But, note would apply say to someone upgrading from 12.0.0 to 12.1.1. They could be not aware of (miss to read) release notes in 12.1.0 and then be surprised. Chance is of customer missing being aware of 12.1.0 and it's release notes is higher due that release being very shortly active.